cypress/e2e/documents.spec.js

```diff
@@ -4,10 +4,17 @@ describe('Model Plan Documents', () => {
     cy.visit('/');
   });
 
-  it('uploads and removes a document', () => {
-    cy.clickPlanTableByName('Empty Plan');
+  it('uploads document through the Documents Card', () => {
+    cy.visit('/');
+
+    cy.get(`[data-testid="${'table'}"] a`).contains('Empty Plan').click();
+    cy.url().should('include', '/collaboration-area');
+
+    cy.get('[data-testid="page-loading"]').should('not.exist');
```

Comment:
Might be good to wait till my PR is merge, I created an additional cy util to navigate to collab area